src/shader_recompiler/frontend/translate/translate.cpp

```diff
@@ -149,6 +149,8 @@ T Translator::GetSrc(const InstOperand& operand) {
         } else {
             return m0_value;
         }
+    case OperandField::Undefined:
```

Comment:
What is the purpose of this

Reply:
There was a game that went to 'UNREACHABLE' because it was called that, but it was in the middle of several modifications, maybe I shouldn't include that